src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-style: book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-weight: 300;
+}
+
+form[data-qa="big"] {
+  box-sizing: border-box;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+  padding: 0;
+  color: #3d4e61;
+}
+
+form[data-qa="big"]:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  color: #3d4e61;
+}
+
+form[data-qa="big"]:focus {
```

Comment:
This rule is targeting the whole form instead of the input itself. Also, use class for styling